### ISSUE TITLE
From where does this so libs come from ?

### ISSUE THREAD

**Issue by hannesa2** [jlivingstonsg/RoboCam#1]
I can't run because of missing libraries. 

<img width="318" alt="image" src="https://user-images.githubusercontent.com/3314607/163707579-404095ca-ef8e-481a-a04e-dc6d944ec71e.png">

Any idea from where does this come from ?

**Comment by jlivingstonsg** [comment]
This is the original app.
https://play.google.com/store/apps/details?id=ru.proghouse.robocam&hl=sv&gl=US
This is devs webpage and email
http://www.proghouse.ru
admin@proghouse.ru
and this is info on how to code it.
http://www.proghouse.ru/tags/tutorials-on-using-programs
Translate with google translate.
If you get an answer please let me know.   :)  
If you plan to code a robocam app from scratch then please give me link to your github repository or anything.  :)  
........

**Comment by jlivingstonsg** [comment]
> I can't run because of missing libraries.
> 
> <img alt="image" width="318" src="https://user-images.githubusercontent.com/3314607/163707579-404095ca-ef8e-481a-a04e-dc6d944ec71e.png">
> 
> Any idea from where does this come from ?

Read this.
https://stackoverflow.com/questions/56824557/what-is-the-difference-between-armeabi-v7a-arm64-v8a-x86

Regards MagI

**Comment by hannesa2** [comment]
> https://stackoverflow.com/questions/56824557/what-is-the-difference-between-armeabi-v7a-arm64-v8a-x86

Ok, I already know the diff. the question was, from where the code of them comes from.
But I've no a little bit more clear picture of it. In my point of view, a non-compile-able and obfuscated code makes not that much sense for me. There are several repos, which are premising on the first view, but non of them are working for me.
I'll continue to investigate ....

**Comment by jlivingstonsg** [comment]
> > https://stackoverflow.com/questions/56824557/what-is-the-difference-between-armeabi-v7a-arm64-v8a-x86
> 
> Ok, I already know the diff. the question was, from where the code of them comes from. But I've no a little bit more clear picture of it. In my point of view, a non-compile-able and obfuscated code makes not that much sense for me. There are several repos, which are premising on the first view, but non of them are working for me. I'll continue to investigate ....

This code compile in version Android Studio 4.0 RC1
Are you trying to code a RoboCam app?
If so then I am interested  :)

Regards Magnus 
------------------------

**Comment by hannesa2** [comment]
Yes, it builds, but unfortunately my device is arm64 and my laptop is an Apple Silicon M1 arm64.
But arm64 so libraries are missing, that's why I need to build them by myself. 
<img width="1408" alt="image" src="https://user-images.githubusercontent.com/3314607/164153799-d1ac1ed1-e71a-4793-8575-f58536fa9f5c.png">

This was my initial question

Btw, on my side it builds with Android Bumblebee as a Gradle Project

**Comment by hannesa2** [comment]
For me something like this https://stackoverflow.com/a/14432887/1079990 looks more promising
